eo-maven-plugin/src/main/java/org/eolang/maven/hash/ChsAsText.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2016-2023 Objectionary.com
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included
+ * in all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NON-INFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+package org.eolang.maven.hash;
+
+import com.jcabi.log.Logger;
+import java.io.IOException;
+import java.net.URL;
+import org.cactoos.Text;
+import org.cactoos.text.TextOf;
+import org.cactoos.text.UncheckedText;
+
+/**
+ * Commit hashes table as text from objectionary.
+ *
+ * @since 0.29.6
+ */
+public final class ChsAsText implements Text {
```

Comment:
@maxonfjvipon By using this name you hide the information about source of the Commit Hashes. First time I though that it is loaded from a filesystem or memory.

Moreover when you are using prefixed names it is assumed that all objects with the same prefix implement the same interface which isn't you case:
https://www.yegor256.com/2020/03/03/prefixed-naming.html

Reply:
@volodya-lombrozo I don't see an elegant way to unite these classes under one interface. WDYT about the names: `CommitHashesAsMap` and `CommitHashesAsText`

Reply:
@maxonfjvipon They look pretty nice to me. However, maybe we can remove `as`: `CommitHashesMap` and `CommitHashesText `?